wiki/Shortcut_key_reference/en.md

```diff
@@ -247,7 +247,7 @@ These shortcuts work anywhere within the beatmap editor:
 | `Ctrl` + `Shift` + `F` | Convert the selected slider into a hit circle stream (opens a dialog). |
 | `Ctrl` + `Left/Right/Up/Down Arrow` | Nudge the selected notes in the pressed direction. (osu!standard, osu!taiko, osu!catch) |
```

Comment:
```suggestion
 | `Ctrl` + `Left/Right/Up/Down Arrow` | Nudge the selected notes in the pressed direction. (osu!, osu!taiko, osu!catch) |
```

Reply:
since there are 8 occasions of this, it's probably good for a separate PR instead